src/styles/card.scss

```diff
@@ -0,0 +1,96 @@
+@import "./utils/variables";
+@import "./stars";
+
+.card {
+  box-sizing: border-box;
```

Comment:
it is a common practice to apply `box-sizing: border-box` for all elements via `*` selector